.github/workflows/ci.yml

```diff
@@ -126,6 +127,96 @@ jobs:
             ${{ env.CHAIN_SPEC }}.json
             ${{ env.CHAIN_SPEC }}.json.state
 
+  integration-test:
+    #      name: ${{ matrix.test }}
+    name: integration-test
+    runs-on: ubuntu-20.04
```

Comment:
everywhere else we use 22.04. Might be worth to bump?

Reply:
I was wondering about that. How about Paynuq: do they build the collator themselves, or would it be an obstacle if we switch the build os?